packages/python-stem/PKGBUILD

```diff
@@ -0,0 +1,26 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname='python-stem'
+pkgver=1849.c320541
+pkgrel=1
+pkgdesc='A controller library for Tor.'
+arch=('any')
+url='https://stem.torproject.org/'
+license=('LGPL3')
+depends=('python')
+makedepends=('git' 'python-setuptools')
+source=('git://git.torproject.org/stem.git')
+sha1sums=('SKIP')
+
+pkgver() {
+  cd "$srcdir/stem"
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
```

Comment:
While you're at it, you can also add a build step.

Reply:
Added.